distribution/docs/src/main/resources/content/_appReferences/mg-spatial.adoc

```diff
@@ -9,29 +9,26 @@ The ${ddf-spatial} Application provides KML transformer and a KML network link e
 ==== Offline Gazetteer Service
 
 In the ${ddf-spatial} Application, the `offline-gazetteer` is installed by default.
-This feature enables you to use an offline index of GeoNames data (as an alternative to the GeoNames Web service enabled by the `webservice-gazetteer` feature) to perform searches via the gazetteer search box in the Search UI.
+This feature enables you to use an offline source of GeoNames data (as an alternative to the GeoNames Web service enabled by the `webservice-gazetteer` feature) to perform searches via the gazetteer search box in the Search UI.
 
-By default a small index set is included with the offline gazetteer. This index can be expanded or updated by using the `geonames:update` command.
+By default a small set of GeoNames data is included with the offline gazetteer. The GeoNames data is stored as metacards in the core catalog and are tagged with `geonames` and `gazetteer`. This collection of GeoNames metacards can be expanded or updated by using the `gazetteer:update` command.
```

Comment:
❓ Should we be using `${ddf-catalog}` wherever we are using the word `catalog`?

Reply:
I think Solr is an implementation detail that we need not include here

Reply:
We use ${ddf-catalog} in other places but not sure exactly when it should be used. Seems like we should use it here. @ricklarsen, does that sound right?

Reply:
I think the difference is the catalog itself vs the catalog application. The ${property} is for the application.